### PR TITLE
Datatables select extension

### PR DESCRIPTION
**Pull request by tjveldhuizen**
I've added the [Select](https://datatables.net/extensions/select/) extension

### CODE REVIEW

File: Resources/doc/extensions.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 1. [Buttons](#1-buttons)
 2. [Responsive](#2-responsive)
+2. [Select](#3-select)
```
Comment (Seb33300) [inline Resources/doc/extensions.md]: 3. ?

File: Resources/doc/extensions.md [hunk 2]
```diff
@@ -218,3 +219,66 @@ With the Responsive class you can set some `display` options to define how the h
...
+    // ...
+
+    $this->extensions->set(array(
+        'blurable' => false,
+        'className': 'selected',
+        'info': true,
+        'items': 'rows',
+        'selector': 'td, th',
+        'style': 'os',
+    ));
```
Comment (Seb33300) [inline Resources/doc/extensions.md]: I didn't work on others extensions, but if I compare this part with [buttons](https://github.com/stwe/DatatablesBundle/blob/master/Resources/doc/extensions.md#advanced-example) and [responsive](https://github.com/stwe/DatatablesBundle/blob/master/Resources/doc/extensions.md#advanced-example-1) extensions, options have to be embedded in an array with the name of the extension as the key of the array.

So this should like like:
```
$this->extensions->set(array(
    //'select' => true,
    'select' => array(
        'blurable' => false,
        'className': 'selected',
        'info': true,
        'items': 'rows',
        'selector': 'td, th',
        'style': 'os',
    ),
));
```

File: Resources/views/datatable/extensions.html.twig [hunk 3]
```diff
@@ -68,3 +68,14 @@
         },
     {% endif %}
 {% endif %}
+
+{# Buttons Extension #}
```
Comment (Seb33300) [inline Resources/views/datatable/extensions.html.twig]: Select Extension ?

File: Resources/doc/extensions.md (hunk 2 above)
Comment (tjveldhuizen) [reply Resources/doc/extensions.md]: OK, apparently I was sleeping while updating the documentation. I'll fix it.

**Comment by Seb33300** [comment]
All indentations of extensions.html.twig have been modified with your Template fix commit :(
This look like you have modified all the file

**Comment by tjveldhuizen** [comment]
I'm sorry, I don't understand why my IDE is changing indentation while doing the commit... have to look into it again

**Comment by eco3** [comment]
When will this implemented in the master branch?
I'm really in need of this extension.

**Comment by stephanvierkant** [comment]
Great work!

**Comment by stephanvierkant** [comment]
@Seb33300 I'm using this extension and can confirm it works perfectly. Can you merge this?

**Pull request merged**